mac/KeymanEngine4Mac/KeymanEngine4Mac/KME/KMBinaryFileFormat.h

```diff
@@ -189,6 +189,8 @@ struct COMP_KEY {
 #define KVKS_RCTRL              0x10
 #define KVKS_LALT               0x20
 #define KVKS_RALT               0x40
+//#define KVKS_META               0x80    // Either Meta-key flag (tentative).  Not usable in keyboard rules;
+                                          // Used internally to ensure Meta-key shortcuts safely bypass rules
```

Comment:
This is for the On Screen Keyboard (KVK), not actually the same as the KMX definitions. 
```suggestion
```

You want to add the definition around line 120.

Reply:
Huh.  Don't know how I managed to gloss over that section the first time.  Good catch.